src/providers/staleness-provider.js

```diff
@@ -24,9 +24,9 @@ import { OBJECT_TYPES, STALENESS_STATUS_MAP } from '../const.js';
 import { buildStalenessResponseObject } from '../utils.js';
 
 export default class YamcsStalenessProvider {
-    #openmct;
-    constructor(openmct, latestTelemetryProvider) {
-        this.#openmct = openmct;
+    constructor(openmct, realtimeTelemetryProvider, latestTelemetryProvider) {
```

Comment:
we should remove the `openmct` parameter as it's unused

Reply:
Good catch. done.